js/common.js

```diff
@@ -1956,10 +1955,13 @@ function setupAdaptDropdown(config)
         },
         templateResult: config.templateresult,
         templateSelection: config.templateselection,
-    })
-        .bind('setValue', function (e, value) {
-            $('#' + field_id).val(value).trigger('change');
-        });
+    };
+    if (config.placeholder !== undefined && config.placeholder !== '') {
+        options.placeholder = config.placeholder;
+    }
+    const select2_el = $('#' + field_id).select2(options).bind('setValue', (e, value) => {
```

Comment:
Not a blocker, but splitting instructions is easier to read IMHO.

```suggestion
    const select2_el = $('#' + field_id).select2(options);

    select2_el.bind('setValue', (e, value) => {
```